packages/app/src/entities/position/context/provider.tsx

```diff
@@ -66,12 +66,10 @@ export function PositionProvider({ children }: { children: JSX.Element }) {
       [],
     )
 
-    if (lendPositions.length || debtPositions.length) {
-      setPosition({
-        deposits: lendPositions,
-        debts: debtPositions,
-      })
-    }
+    setPosition({
```

Comment:
If your lendPositions.length and debtPositions.length are equal to 0 you need setPosition(undefined)

Reply:
fixed